resources/validations/src/ssp.sch

```diff
@@ -209,9 +209,9 @@
         <sch:let name="corrections" value="lv:correct($ok-values, $sensitivity-level)"/>
         <sch:assert role="fatal" id="no-registry-values" test="count($registry/f:fedramp-values/f:value-set) > 0"
             >The registry values at the path '<sch:value-of select="$registry-href"/>' are not present, this configuration is invalid.</sch:assert>
-        <sch:assert role="fatal" id="no-security-sensitivity-level" test="$sensitivity-level != ''">No sensitivty level found, no more validation processing can occur.</sch:assert>
-        <sch:assert role="fatal" id="invalid-security-sensitivity-level" test="empty($ok-values) or not(exists($corrections))"
-            ><sch:value-of select="./name()"/> is an invalid value of '<sch:value-of select="lv:sensitivity-level(/)"/>', not an allowed value of <sch:value-of select="$corrections"/>. No more validation processing can occur.
+        <sch:assert role="fatal" organizational-id="section-c.1.a" id="no-security-sensitivity-level" test="$sensitivity-level != ''">[Section C Check 1.a] No sensitivty level found, no more validation processing can occur.</sch:assert>
```

Comment:
I decided to make use of `@organizational-id` instead of comments because it will get stripped in the XSLT transformation, @brianrufgsa, but for now, it allows stakeholders to map the README Markdown file checklist to the items here more clearly.

Reply:
Per conversation elsewhere I think we are ok on this for now.